### ISSUE TITLE
Remove dead projects

### ISSUE THREAD

**Issue by joshuapassos** [fsprojects/awesome-fsharp#122]
Some projects without recent commits. (for over 1 year)
No make sense have dead projects repositories

**Comment by eugene-g** [comment]
Makes sense, but it depends. Sometimes inactive doesn't mean that it's not awesome :)
@joshuapassos could you provide a list for review?

**Comment by voronoipotato** [comment]
## archived (could be moved to an archive.md for historical/research purposes.)
https://github.com/fsprojects-archive/zzarchive-Cricket
https://github.com/fsharp/zarchive-fsharpbinding
https://github.com/giraffe-fsharp/giraffe.tasks
https://github.com/fsharp/zarchive-fsharpbinding
https://github.com/fsprojects-archive/zzarchive-VisualFSharpPowerTools
https://github.com/flexsearch/flexsearch
https://github.com/fsprojects-archive/zzarchive-ApiaryProvider
https://github.com/fsprojects-archive/zzarchive-FSharp.Interop.ComProvider
https://github.com/fsprojects-archive/zzarchive-FSharp.Data.DbPedia
https://github.com/fsprojects-archive/zzarchive-FSharp.Data.HiveProvider
https://github.com/fsprojects-archive/zzarchive-RestProvider
https://github.com/xyncro/freya
(freya dotnet template) 
https://github.com/vbfox/FSharpTemplates
http://www.tryfsharp.org/

## moved...

https://github.com/fsharp/fsharp -> https://github.com/dotnet/fsharp
https://github.com/kunjee17/awesome-fable -> https://fable.io/resources.html

**Comment by ForNeVeR** [comment]
Thank you, folks. Finally, this has been processed. We now have a basic CI automation to check dead links, and I've also reviewed and sorted through the list you provided.
